docs/front-end/web-accessibility.md

```diff
@@ -0,0 +1,102 @@
+
+- How to Meet [WCAG](https://www.w3.org/WAI/WCAG22/quickref/?versions=2.1) (Quick Reference)
+- Index of [ARIA Design Pattern](https://www.w3.org/WAI/ARIA/apg/) Examples
+
+Other links:
+
+- [W3C WAI Tutorials](https://www.w3.org/WAI/tutorials/)
+- [MDN Accessibility guide](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/HTML])
+- [WebAIM Articles](https://webaim.org/articles/)
+- [A11yProject](https://www.a11yproject.com/)
+- [HTML5 Accessibility](https://www.html5accessibility.com/index.html)
+- [Inclusive Components](https://inclusive-components.design/)
+
+## Tools
+
+In addition to keeping up to date with constantly evolving WCAG guidelines, instances or considerations around ensuring proper color contrasts to implement keyboard navigation and screen reader compatibility can be an extensive list. This is where tools can be a win to simplify the entire process. These tools assist in automating a lot of the heavy lifting, like scanning for common accessibility issues and suggesting possible fixes. It's a good balance between saving time, addressing accessibility related issues as well educating about best practices in accessibility. Some of the tools widely used in the community:
+
+- [IBM Accessibility Checker](https://www.ibm.com/able/toolkit/tools/)
+This is the tool offered by IBM. You can find a web version (chrome and firefox extensions) and the [npm](https://www.npmjs.com/package/accessibility-checker) library on the site. The tool uses IBM's accessibility rules engine detecting WCAG 2.1 accessibility issues for web-based content or applications. Checkout [IBM Accessibility Central](https://pages.github.ibm.com/IBMa/able/) to better understand accessibility and making software accessible at IBM.
+
+IBM's Developer [guidelines and checklist](https://www.ibm.com/able/guidelines/index.html).
+
+- [Tools List by W3C](https://www.w3.org/WAI/test-evaluate/tools/list/)
+
+- [Pa11y](https://pa11y.org/)
+
+- [Axe](https://github.com/dequelabs/axe-core)
+- Using Lighthouse with hints in Chrome inspector
+
+Other than the automated tools, manual testing methods are also used for accessibility testing. Following methods are used after a first pass with automated tools is completed.
+
+- Content Review
+Navigating and reading through content with accessibility best practices in mind.
+
+- Keyboard Testing
+Checking if all interactive elements can be operated with a keyboard.
+
+- Screen Reader Review
+Using a screen reader to test and uncover issues with reading order and interactive elements.
+
+Recommendations of manual accessibility testing tools:
+
+- [Wave](https://wave.webaim.org/)
+- [BrowserStack](https://www.browserstack.com/accessibility-testing)
```

Comment:
Another good tool that you might add here is the [HeadingsMap](https://chromewebstore.google.com/detail/headingsmap/flbjommegcjonpdmenkdiocclhjacmbi) extension.

It allows users to quickly see an outline of the page's heading levels and HTML5 tags. If someone skipped a heading level (eg: went from h2 to h4), this tool helps identify it.